### PR TITLE
13 movement enforced by the environment

### PR DESCRIPTION
**Pull request by alexmorten**
https://trello.com/c/rhqbjI1Z/13-movement-enforced-by-the-environment

### CODE REVIEW

File: cis_cell.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+def is_alive(cell):
+    starvation_threshold = conf.ENERGY_THRESHOLD
+    if cell.energy_level < starvation_threshold:
+        return False
+    else:
+        return True
+
+
+def divide(cell):
```
Comment (alexmorten) [inline cis_cell.py]: `divide` either returns a new cell or `None`, can we change the name to reflect that? Sth. along `maybe_divide` (also not great but you get the idea)?

File: cis_env.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        conf.WORLD_DIMENSION[1] / 2,
+        conf.WORLD_DIMENSION[2] / 2
+    ]),
+    0.002
+)
+
+
+def enforce_movement(cell):
+    global curl
+    new_pos = curl.get(cell.pos.x, cell.pos.y, cell.pos.z)
```
Comment (alexmorten) [inline cis_env.py]: `new_pos` is misleading here, it is actually just a delta, right?

File: cis_cell.py (hunk 1 above)
Comment (Animuslupus) [reply cis_cell.py]: Do you think it would more sense to outsource the check for the energy level into `cell_computing_service.py`?

Comment (alexmorten) [reply cis_cell.py]: I have no strong preference here

**Pull request closed without merging**